pkg/fileutil/fileutil.go

```diff
@@ -57,6 +59,16 @@ func ReadDir(dirpath string) ([]string, error) {
 	return names, nil
 }
 
+// MkdirAllIfNotExist is simliar to os.MkdirAll, but will not return ErrExist
+// if the dir exsits before.
+func MkdirAllIfNotExist(dir string) error {
```

Comment:
Unwieldy function name-- at first glance I expected it to fail if the directory exists. Possible alternatives (not necessarily good): `MkdirAllClobber`, `MkdirAllAlways`, `TouchDirAll`